birdhouse/config/magpie/magpie.ini.template

```diff
@@ -49,6 +49,37 @@ ziggurat_foundations.session_provider_callable = magpie.models:get_session_calla
 github.client_id=${GITHUB_CLIENT_ID}
 github.client_secret=${GITHUB_CLIENT_SECRET}
 
+# temporary token definition (defaults below if omitted)
+# note: token here refers to uuids employed in temporary URL endpoints, not security auth tokens
+magpie.token_expire = 86400
```

Comment:
I am guessing the units for `magpie.token_expire` are seconds?  Can you add a note anyways?

Reply:
Done!